lib/test-ast.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const fs = require('fs');
+const AST = require('recast');
+
+/* Lives throughout one test run as `watch` mode is not supported */
```

Comment:
This module is loaded in a subprocess, so this comment does not apply.

Still, I wonder if we should do the AST parsing explicitly and cache the results based on the file contents.

Reply:
OK, I think I follow you now. My thought regarding caching was not having to read file contents from disk for the _same_ file multiple times, if multiple locations in a file is selected. IO is costly, right? But even if you were to select say 10 locations, it might not be a problem? 

Anyway, I'll drop the cache. Also, performance optimizations should always be accompanied by benchmarks, right?

Reply:
> My thought regarding caching was not having to read file contents from disk for the _same_ file multiple times, if multiple locations in a file is selected.

Right, fair enough. Though the worker process would only ever load the one file.

> IO is costly, right? But even if you were to select say 10 locations, it might not be a problem?

Probably not. Though is this even called more than once, given that a process is launched for a specific test file?



> Also, performance optimizations should always be accompanied by benchmarks, right?

I guess… benchmarking is hard too! 😵